### ISSUE TITLE
[get,post] not work on auto doc.

### ISSUE THREAD

**Issue by oldfeel** [astaxie/beego#1760]
My code is 
````
// @Title 登录
// @Description 登录
// @Param phone query string true 手机号码
// @Param password query string true 密码
// @router /login [get,post]
````

When I click "Try it out!" button in http://localhost:8001/swagger/swagger-1/#!/mobile/登录, I get 
````
Uncaught SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': 'GET,POST' is not a valid HTTP method.
````
![1](https://cloud.githubusercontent.com/assets/1726280/13555427/24701580-e3fb-11e5-8448-0fb8f1561ba4.jpg)

**Comment by saturn4er** [comment]
http://beego.me/docs/mvc/controller/router.md 
> beego will parse the source code automatically. Notes: it will only be generated under dev mode.

Are you running your app in dev mode?

**Comment by oldfeel** [comment]
@saturn4er Yes, I use [get] or [post] is OK, but the [get,post] not work.

Thanks!

**Comment by saturn4er** [comment]
@oldfeel Kind of the error means that you try to pass invalid request method to xhr.open(). Check it in debugger

> client . open(method, url [, async = true [, username = null [, password = null]]])
Throws a SyntaxError exception if either method is not a valid HTTP method or url cannot be parsed.

**Comment by astaxie** [comment]
@oldfeel could you paste the aotu generate router.

**Comment by oldfeel** [comment]
@saturn4er Thank you. I will be try.

**Comment by oldfeel** [comment]
@astaxie 
code in router.go
````
package routers

import (
	"ActivityManager/controllers"
	"ActivityManager/controllers/admin"
	"ActivityManager/controllers/mobile"

	"github.com/astaxie/beego"
)

func init() {
	beego.AutoRouter(&admin.AdminController{})
	beego.AutoRouter(&controllers.CaptchaController{})
	beego.Include(&controllers.WebController{})

	ns := beego.NewNamespace("/v1",
		beego.NSNamespace("/all",
			beego.NSInclude(
				&controllers.BaseController{},
			),
		),
		beego.NSNamespace("/mobile",
			beego.NSInclude(
				&mobile.MobileController{},
			),
		),
	)
	beego.AddNamespace(ns)
}
````
code in controller.go
````
// @Title 登录
// @Description 登录
// @Param phone query string true 手机号码/18537957930
// @Param password query string true 密码/admin
// @router /login [get]
func (c *MobileController) Login() {
	phone := c.GetString("phone")
	password := c.GetString("password")
	password = ofutils.MD5(password)
	user, err := c.GetValue("select id,name,avatar,sex,high,weight,birthday,phone from user where phone=? and password=?", phone, password)
	if err != nil {
		c.Fail(1, "帐号或密码错误")
	} else {
		user["avatar"] = c.GetImage(user["avatar"])
		user["token"] = c.GetUserToken(user)
		c.Success(user)
	}
}
````

**Comment by astaxie** [comment]
@oldfeel it can works in API both GET and POST. but it can't work in swagger doc. I can reproduce in local. It's a swagger issue. We will fix it in 1.6.2. Because we will upgrade swagger to V2

**Comment by melkus** [comment]
With bee 1.8.3, when I change [get] to [get,post] it generates wrong json and no errors in command line.
Just empty description for this API:
        "/myAPI": {},
